.github/ext-check-executable-files.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+#
+# This file is part of the phpBB Forum Software package.
+#
+# @copyright (c) phpBB Limited <https://www.phpbb.com>
+# @license GNU General Public License, version 2 (GPL-2.0)
+#
+# For full copyright and license information, please see
+# the docs/CREDITS.txt file.
+#
+set -e
+
+root="$1"
+extname="$2"
+path="${root}phpBB/ext/${extname}/"
+
+# Check the permissions of the files
+
+# The following variables MUST NOT contain any wildcard
+# Directories to skip
+directories_skipped="-path ${path}develop -o -path ${path}vendor"
```

Comment:
This is false positive because it's finding executable files here `./phpBB/ext/phpbb/ads/.git/description`

Reply:
Works now!!